test/conformance/kuttl/validate/clusterpolicy/standard/exclude/exclude-namespace/README.md

```diff
@@ -0,0 +1,9 @@
+## Description
+
+This test creates a policy to validate all resources have a `foo: bar` label, excluding the `default` namespace.
```

Comment:
```suggestion
This test creates a policy to validate all resources have a `foo: bar` label.
The policy matches on a wildcard but excludes a whole Namespace.
The net result should be any Namespaced resource in the excluded Namespace should not be processed.
```

Reply:
@chipzoller is the change suggested above ok ?

Reply:
Yes, looks better.